experimental/dds/tree2/src/feature-libraries/treeTextCursor.ts

```diff
@@ -45,14 +49,33 @@ import { CursorAdapter, singleStackTreeCursor } from "./treeCursorUtils";
 /**
  * Create a cursor, in `nodes` mode at the root of the provided tree.
  *
- * @returns an {@link ITreeCursorSynchronous} for a single {@link JsonableTree}.
+ * @returns an {@link ITreeCursorSynchronous} in nodes mode for a single {@link JsonableTree}.
+ * @remarks
+ * Do not confuse this with {@link JsonableTree} with the JSON domain:
+ * this takes in data in a specific format that is json compatible (except for FluidHandel values).
```

Comment:
```suggestion
 * this takes in data in a specific format that is json compatible (except for FluidHandle values).
```

Reply:
What do we do in the FluidHandle case? Is that still TBD?

Reply:
For fluid handles, we include IFluidHandles (which is clear from the JsonableTreeType if you look at what it allows). This is simply called out here as they are not actually json compatible, so the name is a bit inaccurate.